### PR TITLE
Updated configuration for access to SoftIron storage network

### PR DESCRIPTION
**Pull request by oneswig**

### CODE REVIEW

File: etc/kayobe/inventory/host_vars/ethsw-b17-u40-41 [hunk 1]
```diff
@@ -59,6 +61,42 @@ switch_interface_config:
...
+          ' exit'] }}
+  Te1/4/5/4:
+    description: softiron-switch-lag-xg28
+    config: >
+      {{ switch_interface_config_lag_member +
+         ['port-channel-protocol lacp',
+          ' port-channel 2 mode active',
+          ' exit'] }}
+  # SoftIron/Ceph Storage network link to dev cluster
+  Hu1/4/6:
```
Comment (markgoddard) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: Is there a 100G link on ethsw11-ra12?

File: etc/kayobe/inventory/host_vars/ethsw-b17-u42 [hunk 2]
```diff
@@ -126,26 +97,24 @@ switch_interface_config:
...
-    config:
-      - "no shutdown"
-      - "switchport"
+  # This switch does routing between VLAN 4 and VLAN 60
   vlan 4:
     description: softiron-ceph
     config:
       - "no shutdown"
+      # Trunk link to BDN switch that hosts the SoftIron uplink
       - "tagged Fo1/4"
```
Comment (markgoddard) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u42]: Should this be 'tagged Fo1/30'?

File: etc/kayobe/inventory/host_vars/ethsw-b17-u40-41 (hunk 1 above)
Comment (oneswig) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: Oh yes - ethsw1-ra12 Eth 1/11 is a 100G link between dev and alaska for Ceph (VLAN 4)

Comment (oneswig) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: But there's also a separate link for VLAN 60 between these racks, which predates it...

File: etc/kayobe/inventory/host_vars/ethsw-b17-u42 (hunk 2 above)
Comment (oneswig) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u42]: Quite so, good spot.

**Pull request merged**